services/roomService/src/types/Player.ts

```diff
@@ -23,6 +26,7 @@ export default class Player {
     };
     this._userName = userName;
     this._id = nanoid();
+    this._townMessageChain = new MessageChain;
```

Comment:
shouldn't this be `new MessageChain()` or is the linter removing those parens?

the parens make it more clear it's an instantiation, it's pretty weird in my mind not to have this